backup_and_restore/application_backup_and_restore/backing_up_and_restoring/restoring-applications.adoc

```diff
@@ -6,9 +6,9 @@ include::_attributes/common-attributes.adoc[]
 
 toc::[]
 
-You restore application backups by creating a xref:../../../backup_and_restore/application_backup_and_restore/backing_up_and_restoring/restoring-applications.adoc#oadp-creating-restore-cr_restoring-applications[`Restore` custom resources (CRs)].
+You restore application backups by creating a `Restore` custom resource (CR) by using the OpenShift API for Data Protection (OADP). See xref:../../../backup_and_restore/application_backup_and_restore/backing_up_and_restoring/restoring-applications.adoc#oadp-creating-restore-cr_restoring-applications[Creating a Restore CR].
```

Comment:
Do you really create a CR by using the API? I'm not sure of the right wording. 

`creating a `Restore` custom resource (CR) that uses...`??